content/shmem_query_initialized.tex

```diff
@@ -0,0 +1,30 @@
+\apisummary{
+Returns the initialized status of the \openshmem library.
+}
+
+\begin{apidefinition}
+
+\begin{Csynopsis}
+void @\FuncDecl{shmem\_query\_initialized}@(int *initialized);
+\end{Csynopsis}
+
+\begin{apiarguments}
+\apiargument{OUT}{initialized}{Nonzero if the \openshmem library is in the initialized state. Nonzero otherwise.}
+\end{apiarguments}
+
+\apidescription{
+    The \FUNC{shmem\_query\_initialized} call returns the initialization status
+    of the \openshmem library. If the application has called an \openshmem
+    initialization routine and has not yet made the corresponding call to
```

Comment:
What if the call to `shmem_finalize` has been made, but hasn't completed yet.  Should this say the call to `shmem_finalize` has completed or something similar?